### ISSUE TITLE
[Survival][Suggestion] New Resources for Survival 

### ISSUE THREAD

**Issue by Crimson-Artist** [tsunamayo/Starship-EVO#5920]
As I look over the crafting recipes I notice that nearly everything we can craft requires at least 50-100% iron. I understand that in terms of scale keeping everything based around one resource helps with mass production however if theres going to be a sense of progress and a reason to explore the galaxy we will need many more resources to both acquire and craft. I've come up with a short list of new resources or new uses of current resources that could help add some diversity to the crafting system.

New Raw Ore
- Silicon- Having a dedicated ore for it helps cut down on the confusion for most players. It would be nearly as plentiful as iron. 
- Magnesium- A semi common resource for creating early explosives. 
- Sulfur- A resource that should only be found on volcano planets (when they are added). could be used for more advanced explosives such as missiles when we get them.
- Ice- Found only on asteroids, Ice planets, and moons. They are a solid form of water and can be thawed using either a smelter or a new crafting block to give us liquid water.
- Dirt- Acquired from mining normal non rocky ground. Can be used to make earthy decorative blocks like clay vases dirt/mud building blocks. May also be used in the potential farming system.
- Uranium- Found only in highly radioactive environments. Could be used as the basis for nuclear power or maybe weapons.
- Iridium- A rare resource. could be used for anything heat based systems like weapons.

New Non-Solids
- Helium- Gathered in planetary rings.
- Methane- Gathered in planetary rings.
- Nitrogen- Gathered in planetary rings.

New Intermediates
- Charcoal- Made from smelting wood. Acts as a craftable form of coal.
- Ceramic- Made from dirt. Can be used for decorative blocks or used in player equipment.

Non-Solids as Crafting ingredients
- Each of the non-solids that currently exist and potentially will exist can be used to make new items using a new crafting block called a refinery. This Refinery will be a kind of hybrid of a assembler and a electrolyser. Both solids and non-solids will be able to be piped in to create new blocks. The most basic crafting recipe will be making itemized versions of each non-solid such as a Water Container, which is a item that can be placed in cargo boxes and used by the assembler as a crafting ingredient.

**Comment by asanagisae** [comment]
I like this idea.

Care must be taken to ensure that players aren't blocked from at least progressing to warp jumps because their starter system is missing a key resource.

One possibility is the inclusion of "optional ingredients" in recipes. For example, you can build a hull block normally, or you can add iridium to the recipe to add extra heat capacity to the block (perhaps at the cost of reducing its contributed ship HP). While common materials are enough to progress to the end game, you will want large amounts of system-specific resources to tune your ship towards a specific purpose.

Games like Factorio and Foundry, which only require a very small collection of base ingredients for a significant portion of the early game (iron, copper, stone, coal), simulate progression through the use of intermediate recipes and crafting buildings. It's a successful and proven strategy.

**Comment by tsunamayo** [comment]
@Crimson-Artist Yes I agree with that and I had in mind most of the stuff proposed. I will add plenty of new resources as I add features to the game (weapon, new power sources)
